src/content/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data.mdx

```diff
@@ -61,10 +58,9 @@ If you're logged in but can't find an expected account or UI page, it may be bec
 * If you have multiple user records (for example, access to more than one New Relic organization), you can [switch between your user records](/docs/accounts/accounts-billing/account-setup/multiple-logins-found).
 * If you can't find an expected account, it may be because you haven't been assigned access to that account. See [user permissions issues](#user-permissions).
 * If you check the account switcher and can't see the account you're looking for, it may be for one of these reasons:
-  * You may need to be added to that account. How you do this depends on your user model: [Our newer user model](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/#account-access) \| [Our original user model](/docs/accounts/original-accounts-billing/original-users-roles/original-account-structure).
+  * You may need to be added to that account. See [How users access accounts](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/#account-access).
```

Comment:
Passive voice here, maybe something like "You may need an administrator to add you to the account." or something along those lines? Also if you wind up making an edit to this line, the second sentence ends a bit abruptly, maybe add a "for more information" or something generic at the end?

Reply:
@jeff-colucci I've pushed up two revisions. Thanks!